phpstan.neon.dist

```diff
@@ -36,6 +36,11 @@ parameters:
         - '#Call to an undefined method .*::transChoice\(\)\.#'
         - '#Trying to mock an undefined method transChoice\(\) on class Symfony\\Component\\Translation\\Translator\.#'
 
+        # Ignore PHPStan not being able to trace changes in $GLOBALS
+        -
+            message: '#Call to method PHPUnit\\Framework\\Assert::assertSame\(\) with .* will always evaluate to false\.#'
```

Comment:
```suggestion
            message: '#Call to method PHPUnit\\Framework\\Assert::assertSame\(\) with \$GLOBALS.* will always evaluate to false\.#'
```

Limiting the regex will prevent false positives that are not related to `$GLOBALS`.

Reply:
Should we also drop the `path` then?

Reply:
This unfortunately won't work as the message looks like this:
```
Call to method PHPUnit\Framework\Assert::assertSame() with array(array('test.listener.a', 'onGetPageLayout'), array('test.listener.c', 'onGetPageLayout'), array('test.listener.b',            
         'onGetPageLayout')) and array(array('test.listener.c', 'onGetPageLayout')) will always evaluate to false.   
```

Reply:
I see. 🤷🏻‍♂️